database/src/main/java/com/firebase/ui/database/paging/DefaultSnapshotDiffCallback.java

```diff
@@ -25,6 +26,7 @@ public boolean areItemsTheSame(@NonNull DataSnapshot oldItem,
         return oldItem.getKey().equals(newItem.getKey());
     }
 
+    @SuppressLint("DiffUtilEquals")
```

Comment:
Why is this lint suppression added in this PR when nothing else about this file changed?

Reply:
Because it caused an error when I run the command `gradlew assembleDebug proguard-tests:build check`

Reply:
Ok I am fine with this, although I bet if we revert the unrelated changes in this PR, this will go away.

Reply:
Oh! Okay.